### PR TITLE
Add shortcut for Run, Format and Execute buttons.

### PR DESCRIPTION
**Pull request by axiomaticcrab**

### CODE REVIEW

File: app/scripts/chromereload.js [hunk 1]
```diff
@@ -20,3 +20,40 @@ connection.onmessage = function (e) {
...
+    executeOnActiveTab(formatCode);
+  }
+  if (command === "reset-code") {
+    executeOnActiveTab(resetCode);
+  }
+});
+
+function runCode() {
+  document.getElementById("run").click();
+  console.log("runCode executed!");
```
Comment (dancannon) [inline app/scripts/chromereload.js:39]: Could we remove these console.logs?

File: app/scripts/contentscript.js [hunk 2]
```diff
@@ -94,4 +94,8 @@ $(function() {
     'autosave':      true,
     'enableHistory': true
   });
+
+  document.getElementById("run").title = "Press ALT+R to run quickly.";
```
Comment (dancannon) [inline app/scripts/contentscript.js:98]: I don't think ` quickly` is really needed here, it seems a bit redundant? What do you think?

**Review by dancannon — COMMENTED** [review]
Thanks for looking into this, I left a few comments but I cannot see any major issues. 

That being said the playground comes with its own keyboard shortcuts (cmd+enter and shift+enter). I am not sure the new shortcuts are really needed.

**Comment by axiomaticcrab** [comment]
Hello Daniel, thanks for the extension. I didn't know that the playground have the shortcuts available and was have some time to play with it.
Since it has the ability I'm closing the PR.

**Pull request closed without merging**